src/test/storage/table_test.cpp

```diff
@@ -12,63 +12,65 @@
 
 namespace opossum {
 
-// class StorageTableTest : public BaseTest {
-//  protected:
-//   void SetUp() override {
-//     t.add_column("col_1", "int");
-//     t.add_column("col_2", "string");
-//   }
+class StorageTableTest : public BaseTest {
+ protected:
+  void SetUp() override {
+    t.add_column("col_1", "int");
+    t.add_column("col_2", "string");
+  }
 
-//   Table t{2};
-// };
+  Table t{2};
+};
 
-// TEST_F(StorageTableTest, ChunkCount) {
-//   EXPECT_EQ(t.chunk_count(), 1u);
-//   t.append({4, "Hello,"});
-//   t.append({6, "world"});
-//   t.append({3, "!"});
-//   EXPECT_EQ(t.chunk_count(), 2u);
-// }
+TEST_F(StorageTableTest, ChunkCount) {
+  EXPECT_EQ(t.chunk_count(), 1u);
+  t.append({4, "Hello,"});
+  t.append({6, "world"});
+  t.append({3, "!"});
+  EXPECT_EQ(t.chunk_count(), 2u);
+}
 
-// TEST_F(StorageTableTest, GetChunk) {
-//   t.get_chunk(ChunkID{0});
-//   // TODO(anyone): Do we want checks here?
-//   // EXPECT_THROW(t.get_chunk(ChunkID{q}), std::exception);
-//   t.append({4, "Hello,"});
-//   t.append({6, "world"});
-//   t.append({3, "!"});
-//   t.get_chunk(ChunkID{1});
-// }
+TEST_F(StorageTableTest, GetChunk) {
+  t.get_chunk(ChunkID{0});
+  // TODO(anyone): Do we want checks here?
+  // EXPECT_THROW(t.get_chunk(ChunkID{q}), std::exception);
+  t.append({4, "Hello,"});
+  t.append({6, "world"});
+  t.append({3, "!"});
+  t.get_chunk(ChunkID{1});
+}
```

Comment:
You have a nearly perfect coverage, it is only missing the const get_chunk method of Table. 

One way to force calling as a const method is std::as_const.